pyigm/clustering/survey_2d2pcf.py

```diff
@@ -65,6 +70,7 @@ def __init__(self, field):
         self.rdiff  = self.rbinedges[1:] - self.rbinedges[:-1]
         self.tdiff  = self.tbinedges[1:] - self.tbinedges[:-1]
 
+
     def addField(self, new_field, SEP_TOL=0.5*u.deg):
```

Comment:
why `SEP_TOL` is defined with upper case?

Reply:
no particular reason